src/main/java/net/homework/webapp/service/DepartmentService.java

```diff
@@ -0,0 +1,15 @@
+package net.homework.webapp.service;
+
+import net.homework.webapp.dto.DepartmentDto;
+
+import java.util.List;
+
+public interface DepartmentService {
+    List<DepartmentDto> getAllDepartments();
```

Comment:
Services should work with domain objects only

Reply:
fixed